src/app/view/components/side-navigation-menu/side-navigation-menu.component.ts

```diff
@@ -74,7 +79,38 @@ export class SideNavigationMenuComponent implements AfterViewInit, OnDestroy {
     }
   }
 
-  constructor(private elementRef: ElementRef) {}
+  constructor(
+    private elementRef: ElementRef,
+    private getAllMenuSystemMenu: GetAllMenuSystemMenuUsecase
+  ) {}
+  ngOnInit(): void {
+    this._items = navigation.map((item) => {
+      item.path = this.checkPath(item.path);
+      return { ...item, expanded: !this._compactMode };
+    });
+    this.getAllMenuSystemMenu.execute({}).subscribe((grid: PageResultModel<SystemMenuModel>) => {
+      var mappedItems = grid.data?.map((item) => this.itemMenuMap(item));
```

Comment:
```suggestion
      var mappedItems = grid.data?.map(this.itemMenuMap);
```
Você deve conseguir passar a função como parâmetro sem usar a arrow function só para executar outra função.